### ISSUE TITLE
macOS hardened runtime support

### ISSUE THREAD

**Issue by buu700** [apache/cordova-electron#174]
# Feature Request

## Motivation Behind Feature

Additional security.

## Feature Description

The `hardenedRuntime` option at https://www.electron.build/configuration/mac.

## Alternatives or Workarounds

Currently using a fork at https://github.com/buu700/cordova-electron-tmp.

**Comment by timbru31** [comment]
PR's welcome, as it seems you've already implemented the property. :)

**Comment by buu700** [comment]
Ah, well I just have `buildConfigs.hardenedRuntime = true` thrown in basically somewhere random as a temporary workaround, and wouldn't be familiar enough with the project structure offhand to do it properly and make it configurable, but I could potentially put in the time to submit a PR at some point in the future.

**Comment by erisu** [comment]
The option was added and merged in PR: https://github.com/apache/cordova-electron/pull/200

It should have been released with Cordova Electron 3.0.0.

You can upgrade with the following steps:
```
cordova platform remove electron
cordova platform add electron@3.0.0
```

**Comment by buu700** [comment]
Awesome, thanks!
